### PR TITLE
Updates to LOC type RFC1876

### PR DESCRIPTION
**Pull request by systemcrash**
https://www.rfc-editor.org/rfc/rfc1876

### CODE REVIEW

File: types.go [hunk 1]
```diff
@@ -198,7 +198,7 @@ const (
 	_CD = 1 << 4  // checking disabled
 )
 
-// Various constants used in the LOC RR. See RFC 1887.
+// Various constants used in the LOC RR. See RFC 1876.
```
Comment (miekg) [inline types.go:201]: fine with this

File: types_test.go [hunk 2]
```diff
@@ -5,37 +5,37 @@ import (
 )
 
 func TestCmToM(t *testing.T) {
-	s := cmToM(0, 0)
+	s := cmToM((0 << 0) + 0)
```
Comment (tmthrgd) [inline types_test.go:8]: `<< 4`

File: types.go [hunk 3]
```diff
@@ -792,7 +792,10 @@ type LOC struct {
 
 // cmToM takes a cm value expressed in RFC 1876 SIZE mantissa/exponent
 // format and returns a string in m (two decimals for the cm).
-func cmToM(m, e uint8) string {
+func cmToM(x uint8) string {
+	var m = x & 0xf0 >> 4
+	var e = x & 0x0f
```
Comment (tmthrgd) [inline types.go:797]: `:=` rather than `var ... =`

**Review by tmthrgd — CHANGES_REQUESTED** [review]
Minor nits.

**Review by tmthrgd — APPROVED** [review]
LGTM.

File: types_test.go (hunk 2 above)
Comment (systemcrash) [reply types_test.go]: seemed a waste of cpu cycles :)

**Pull request merged**